install.sh

```diff
@@ -79,125 +77,133 @@ function backup_append() {
+	exit 1
 fi
 
 BASH_IT="$(cd "$(dirname "$0")" && pwd)"
 
 case $OSTYPE in
-  darwin*)
-    CONFIG_FILE=.bash_profile
-    ;;
-  *)
-    CONFIG_FILE=.bashrc
-    ;;
+	darwin*)
+		CONFIG_FILE=.bash_profile
+		;;
+	*)
+		CONFIG_FILE=.bashrc
+		;;
 esac
 
 BACKUP_FILE=$CONFIG_FILE.bak
 echo "Installing bash-it"
 if ! [[ $silent ]] && ! [[ $no_modify_config ]]; then
-  if [ -e "$HOME/$BACKUP_FILE" ]; then
-    echo -e "\033[0;33mBackup file already exists. Make sure to backup your .bashrc before running this installation.\033[0m" >&2
-    while ! [ $silent ];  do
-      read -e -n 1 -r -p "Would you like to overwrite the existing backup? This will delete your existing backup file ($HOME/$BACKUP_FILE) [y/N] " RESP
-      case $RESP in
-      [yY])
-        break
-        ;;
-      [nN]|"")
-        echo -e "\033[91mInstallation aborted. Please come back soon!\033[m"
-        exit 1
-        ;;
-      *)
-        echo -e "\033[91mPlease choose y or n.\033[m"
-        ;;
-      esac
-    done
-  fi
-
-  while ! [ $append_to_config ]; do
-    read -e -n 1 -r -p "Would you like to keep your $CONFIG_FILE and append bash-it templates at the end? [y/N] " choice
-    case $choice in
-    [yY])
-      backup_append
-      break
-      ;;
-    [nN]|"")
-      backup_new
-      break
-      ;;
-    *)
-      echo -e "\033[91mPlease choose y or n.\033[m"
-      ;;
-    esac
-  done
+	if [ -e "$HOME/$BACKUP_FILE" ]; then
+		echo -e "\033[0;33mBackup file already exists. Make sure to backup your .bashrc before running this installation.\033[0m" >&2
+		while ! [ $silent ]; do
+			read -e -n 1 -r -p "Would you like to overwrite the existing backup? This will delete your existing backup file ($HOME/$BACKUP_FILE) [y/N] " RESP
+			case $RESP in
+				[yY])
+					break
+					;;
+				[nN] | "")
+					echo -e "\033[91mInstallation aborted. Please come back soon!\033[m"
+					exit 1
+					;;
+				*)
+					echo -e "\033[91mPlease choose y or n.\033[m"
+					;;
+			esac
+		done
+	fi
+
+	while ! [ $append_to_config ]; do
+		read -e -n 1 -r -p "Would you like to keep your $CONFIG_FILE and append bash-it templates at the end? [y/N] " choice
+		case $choice in
+			[yY])
+				backup_append
+				break
+				;;
+			[nN] | "")
+				backup_new
+				break
+				;;
+			*)
+				echo -e "\033[91mPlease choose y or n.\033[m"
+				;;
+		esac
+	done
 elif [[ $append_to_config ]]; then
-  # backup/append
-  backup_append
+	# backup/append
+	backup_append
 elif [[ $silent ]] && ! [[ $no_modify_config ]]; then
-  # backup/new by default
-  backup_new
+	# backup/new by default
+	backup_new
 fi
 
 # Disable auto-reload in case its enabled
 export BASH_IT_AUTOMATIC_RELOAD_AFTER_CONFIG_CHANGE=''
 # Load dependencies for enabling components
-source "$BASH_IT/lib/composure.bash"
+# shellcheck source=./vendor/init.d/composure.bash
+source "$BASH_IT/vendor/init.d/composure.bash"
```

Comment:
this needs to be updated as well

Reply:
Hmm, I missed that :smile: 
Maybe there should be some tests of the install script?